config/jobs/kubernetes-security/generated-security-jobs.yaml

```diff
@@ -339,50 +339,6 @@ presubmits:
           defaultMode: 256
           secretName: ssh-security
     trigger: (?m)^/test( all| pull-security-kubernetes-bazel-build),?(\s+|$)
```

Comment:
uh this file should not be edited since it's auto generated? all security repo jobs should be auto generated ideally...

Reply:
nevermind, weird that this caused it to move...

Reply:
huh? I ran hack/update-config.sh

Reply:
oh, the order of job loading is changed - it loads the main config, then appends the test-infra config, so those canary jobs got moved down

Reply:
(since we don't sort them anymore)